### ISSUE TITLE
Build error - buildStylesheets.mjs

### ISSUE THREAD

**Issue by JanBN** [adazzle/react-data-grid#2162]
Hello I have cloned canary branch, run npm install, then build script and it fails here:

```
> react-data-grid@7.0.0-canary.22 postbuild C:\projects\NewGrid2\react-data-grid
> node tools/buildStylesheets.mjs

internal/modules/cjs/loader.js:975
    throw new ERR_REQUIRE_ESM(filename);
    ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: C:\projects\NewGrid2\react-data-grid\tools\buildStylesheets.mjs
    at Module.load (internal/modules/cjs/loader.js:975:11)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  code: 'ERR_REQUIRE_ESM'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-data-grid@7.0.0-canary.22 postbuild: `node tools/buildStylesheets.mjs`
npm ERR! Exit status 1
```

Can somebody help me with this ?

Thank you

**Comment by JanBN** [comment]
Ok this problem solved by installing latest nodejs, version 12.8.4. But still it is not compiling because of this error:


```
Module parse failed: Unexpected token (102:37)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|   const totalHeaderHeight = headerRowHeight + (enableFilters ? headerFiltersHeight : 0);
|   const clientHeight = height - 2 // border width
>   - totalHeaderHeight - (summaryRows?.length ?? 0) * rowHeight - (totalColumnWidth > viewportWidth ? getScrollbarSize() : 0);
|   const [rowOverscanStartIdx, rowOverscanEndIdx] = getVerticalRangeToRender(clientHeight, rowHeight, scrollTop, rows.length);
|   /**
```

It seems like it can not handle new typescript feature ?? and ?.something.
As a workaround changed reverted back tsconfig - https://github.com/adazzle/react-data-grid/commit/38a81b60fd96d253e01e59d8b8a01e3604668d66#diff-e5e546dd2eb0351f813d63d1b39dbc48

But either way there is some problem, because I would expect when a make a clean clone, run npm install it should be working.

**Comment by amanmahajan7** [comment]
Duplicate issue https://github.com/adazzle/react-data-grid/issues/2091. Please check the [installation guide](https://github.com/adazzle/react-data-grid#install) on how to configure Webpack and babel 7
